packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -38,6 +38,8 @@ class IconButton extends StatelessWidget {
   const IconButton({
     Key key,
     this.size: 24.0,
+    this.padding: const EdgeInsets.all(8.0),
+    this.alignment: FractionalOffset.centerLeft,
```

Comment:
The leading AppBar IconButton now consumes all of the available width (and height).  The icon within the button needs to be left justified (and vertically centered) to appear where it's supposed to.